### ISSUE TITLE
Regarding Results Evaluation 

### ISSUE THREAD

**Issue by connectedhealthcaremy** [stefanonardo/pytorch-esn#1]
we have run your algorithm and got the following results on the sample model 
tensor([[ 0.4779],
        [ 0.2621],
        [ 0.1902],
        ...,
        [ 0.3912],
        [ 0.3592],
        [ 0.3210]])
We do not know how to get the  classification accuracy please guide us we will be really thankful to you

**Comment by stefanonardo** [comment]
I'm going to release a new version next week with many improvements and fixes, including the ability to train the network for a classification task.

**Comment by stefanonardo** [comment]
@connectedhealthcaremy I updated the code (this new version works with PyTorch 0.4 only). On the front page you will notice that I added a parameter for classification tasks, namely `output_steps`.

Before this update, you had to pass a target value for each timestep of the timeseries, i.e. size had to be (timesteps, batch, feature). Now, if you pass 'mean' or 'last' to  `output_steps`, your target's size must be (1, batch, feature).
